Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftSecurityEventFirewall.yaml

```diff
@@ -0,0 +1,173 @@
+Parser:
```

Comment:
can we do data and filtering test using ASC portal?

Reply:
YES